### PR TITLE
PREOPS-3728: implement the Spanish translation of the glossary & acronyms

### PR DESCRIPTION
**Pull request by plazas**

### CODE REVIEW

File: etc/glossarydefs_es.csv [hunk 1]
```diff
@@ -1,24 +1,1576 @@
...
+AGN,Núcleos activos de galaxia
+AGU,Unión Estadounidense de Geofísica
+AI,Inteligencia artificial
+AIC,Criterio de información de Akaike
+AIP,Instituto Estadounidense de Física
+AISES,Sociedad Indígeno-Estadounidense de Ciencia e Ingeniería
+AIT,"Ensamblaje, integración y prueba"
+AI&T,"Ensamblaje, integración y prueba"
+AIV,"Ensamblaje, integración y verificación"
+AJ,The Astronomical Journal [Revista de Astronomía]
```
[the remark this answers is not in the document]
Comment (plazas) [reply etc/glossarydefs_es.csv:45]: I guess "Journal" is being translated here as "Revista". I'll add the definite article "La".

File: etc/glossarydefs_es.csv [hunk 2]
```diff
@@ -1,24 +1,1576 @@
...
+CEP,Plan de estimación de costos
+CEPP,Plan de prevención de la exposición al COVID-19
+CERN,Organización Europea para la Investigación Nuclear
+CET,Equipo de interacción comunitaria
+CfA,Centro de Astrofísica (Harvard-Smithsonian)
+CFD,mecánica de fluidos computacional
+CFHT,Telescopio Canadá-Francia-Hawái
+CFHT-LS,Investigación de imágenes históricas con 5 bandas de paso realizada en el Telescopio Canadá-Francia-Hawái de 2003 a 2008
+CFO,Director Financiero
+CFHTLS,Canada-France-Hawaii Telescope Legacy Survey [Investigación histórica del telescopio Canadá-Francia-Hawái]
```
[the remark this answers is not in the document]
Comment (plazas) [reply etc/glossarydefs_es.csv:159]: I'll delete this, not sure why it was delivered like that.

File: etc/glossarydefs_es.csv [hunk 3]
```diff
@@ -1,24 +1,1576 @@
...
+CFR,Código de Regulaciones Federales
+CHEP,Computación en Física Nuclear y de Altas Energías
+CHIME,Experimento Canadiense de Mapeo de la Intensidad del Hidrógeno
+CI,Integración continua
+CI,Ciberinfraestructura
+CIGALE,Code Investigating GALaxy Emission [Código de investigación de la emisión de galaxias]
```
[the remark this answers is not in the document]
Comment (plazas) [reply etc/glossarydefs_es.csv:165]: Same as above.

File: etc/glossarydefs_es.csv [hunk 4]
```diff
@@ -1,24 +1,1576 @@
...
+XRISM,Misión de espectroscopia e imágenes de rayos X [satélite de la Agencia Japonesa de Exploración Aeroespacial]
+XSD,XML Schema Definition [lenguaje de esquema para documentos XML]
+XSEDE,Entorno Extremo de Descubrimientos en Ciencia e Ingeniería
+XSL,eXtensible Stylesheet Language [lenguaje de programación para páginas web]
+XSLT,eXtensible Stylesheet Language Transformation [estándar para transformación de documentos XML]
+YAML,Otro lenguaje de marcado más [formato de serialización de datos]
+YSO,Objeto estelar joven
+ZD,distancia cenital 
+ZTF,Instalación de [búsqueda de objetos] transitorios de Zwicky 
+accidente,"Evento no deseado que produce un daño a personas o bienes, o una pérdida en procesos.  Los accidentes se producen a partir del contacto con una sustancia o fuente de energía por encima del umbral límite de la estructura del cuerpo."
```
[the remark this answers is not in the document]
Comment (plazas) [reply etc/glossarydefs_es.csv:1163]: I ended up putting the Spanish key in parentheses at the start of the description, matching the key in English.

**Review by womullan — APPROVED** [review]
i think we are good to go

**Pull request merged**